tracing/tracing/src/main/java/io/helidon/tracing/Scope.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Oracle and/or its affiliates.
+ * Copyright (c) 2022, 2024 Oracle and/or its affiliates.
```

Comment:
Huh; not changed, apparently? Maybe revert?

Reply:
Fixed